composer.json

```diff
@@ -29,18 +29,18 @@
     },
     "require-dev": {
         "ext-gmp": "*",
-        "codeception/aspect-mock": "^2.0",
-        "doctrine/annotations": "^1.2",
-        "goaop/framework": "^2.1",
-        "ircmaxell/random-lib": "^1.1",
-        "jakub-onderka/php-parallel-lint": "^0.9.0",
-        "mockery/mockery": "^1.0",
-        "moontoast/math": "^1.1",
-        "php-mock/php-mock-phpunit": "^2.0",
+        "codeception/aspect-mock": "2.1.1",
+        "doctrine/annotations": "1.4.0",
```

Comment:
The versions are the latest working of PHP 7.0 (there are other versions which support 7.1+ only)